armadillo1/DEVELOPMENT.md

```diff
@@ -0,0 +1,84 @@
+# Development and testing of the Armadillo stack
+To test the deployment we are using Vagrant to deploy the ansible playbook locally on your machine. You will need some prerequisites to deploy locally.
+
+* [Vagrant](https://www.vagrantup.com/downloads)
+* [Virtualbox](https://www.virtualbox.org/wiki/Downloads)
+* [git](https://git-scm.com/downloads)
+
+## Creating the configuration
+Create a file called: `Vagrant` with the following configuration:
+
+```javascript
+Vagrant.configure("2") do |config|
+  config.vm.box = "centos/8"
+  config.vm.box_version = "2011.0"
+  config.vm.network "forwarded_port", guest: 80, host: 8080
+  config.vm.provider "virtualbox" do |vb|
+    vb.gui = false
+    vb.customize ['modifyvm', :id, '--graphicscontroller', 'none']
+    vb.customize ['modifyvm', :id, '--audio', 'none']
+  end
+  config.vm.provision "ansible" do |ansible|
+    ansible.limit = "all"
+    ansible.playbook = "playbook.yml"
+    ansible.verbose = false
+  end
+end
+```
+
+Create an file called `playbook.yml` with the content from here: [ansible galaxy content](#creating-playbook.yml).
```

Comment:
instead of _from_ you can use _described_: Create an file called `playbook.yml` with the content described here